flatdata-rs/lib/Cargo.toml

```diff
@@ -13,4 +13,5 @@ edition = "2018"
 [dependencies]
 diff = "0.1.11"
 memmap2 = "0.2.0"
+tar = "0.4"
```

Comment:
Since this adds another dependency, can you make it an optional feature flag for the crate?